apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.styles.ts

```diff
@@ -39,4 +39,15 @@ export const styles = {
     width: '100px',
     margin: 'auto',
   }),
+  note: css({
```

Comment:
nice -  good fix

Reply:
great point! a good reason to have this as a component of its own so we only have to change in one spot moving forward 👍